### ISSUE TITLE
The lowest item level isn't displayed on the cart page

### ISSUE THREAD

**Issue by natalia-muzyka** [ChildMindInstitute/mindlogger-applet-library#50]
**Preconditions**
The site https://library-staging.mindlogger.org/ is opened
The user is logged in

**Steps to reproduce**
1. Add an applet with at least to the cart
2. Proceed to the cart page
3. Expand the activity to the lowest item level
4. Observe the result

**Actual result**
The lowest item level isn't displayed on the cart page

**Expected result**
The applet's levels are displayed the same as on the main page/search results page

**Attachment**
https://images.zenhubusercontent.com/103405139/11ea3991-61de-48ca-9efb-67b95cf4b2de/search_vs_cart_media.mp4
![main vs cart.png](https://images.zenhubusercontent.com/60ab57c8916945b978a3da92/da130b6e-5432-4b07-a27e-d9f30f34c7b8)

**Environment**
https://library-staging.mindlogger.org/
Win 10 // Chrome 91.0.4472.77

**Comment by natalia-muzyka** [comment]
@Sshetty2 The lowest item now can be reached but we missed the ability to expand/collapse the item(to see/hide the item options).
It'd be not comfortable to work with big applets without this ability.

Compare: ![image.png](https://images.zenhubusercontent.com/60ab57c8916945b978a3da92/46debeb5-6951-46e0-9ec7-a815d077fd63)
Video: https://www.screencast.com/t/lUxbSAmaIki

**Comment by natalia-muzyka** [comment]
Created a separate ticket due to my last comment
[The ability to expand/collapse the activity item is missed#72](https://app.zenhub.com/workspaces/mindlogger-webapp-5e11094d0c26311588da9626/issues/childmindinstitute/mindlogger-applet-library/72)

**Comment by natalia-muzyka** [comment]
testing is blocked by [Infinity loader on Applet Library](https://app.zenhub.com/workspaces/mindlogger-webapp-5e11094d0c26311588da9626/issues/childmindinstitute/mindlogger-applet-library/64)

**Comment by natalia-muzyka** [comment]
Verified as fixed:
https://www.screencast.com/t/5pWza6iXf67g

Environment
https://library-staging.mindlogger.org/
Win 10 // Chrome 91.0.4472.114
